src/odata-to-abstract-sql.ts

```diff
@@ -1034,6 +1043,25 @@ export class OData2AbstractSQL {
 			return this.ReferencedField(prop.resource, prop.name);
 		}
 	}
+
+	Method(prop: any): BooleanTypeNodes | { resource: Resource; name: string } {
```

Comment:
`unknown` would be preferable for new code, it should also help make sure you've got all the validation checks you need
```suggestion
	Method(prop: unknown): BooleanTypeNodes | { resource: Resource; name: string } {
```